test/testScripts/lightspeed/testLightspeed.test.ts

```diff
@@ -17,6 +17,7 @@ import { lightSpeedManager } from "../../../src/extension";
 import {
   testInlineSuggestionByAnotherProvider,
   testInlineSuggestionProviderCoExistence,
+  testIgnorePendingSuggestion,
```

Comment:
This test case (`testIgnorePendingSuggestion`) was written for #1113, but it was not executed by mistake. I added it here to be included in e2e tests.